vitest.workspace.ts

```diff
@@ -1,20 +1,32 @@
 import { defineWorkspace } from "vitest/config";
 
+const packagedEmbedTestsOnly = process.argv.includes("--packaged-embed-tests-only");
 // defineWorkspace provides a nice type hinting DX
-export default defineWorkspace([
-  {
-    test: {
-      include: ["packages/**/*.{test,spec}.{ts,js}", "apps/**/*.{test,spec}.{ts,js}"],
-      // TODO: Ignore the api until tests are fixed
-      exclude: ["apps/api/**/*", "**/node_modules/**/*"],
-    },
-  },
-  {
-    test: {
-      name: "@calcom/closecom",
-      include: ["packages/app-store/closecom/**/*.{test,spec}.{ts,js}"],
-      environment: "jsdom",
-      setupFiles: ["packages/app-store/closecom/test/globals.ts"],
-    },
-  },
-]);
+const workspaces = packagedEmbedTestsOnly
+  ? [
+      {
+        test: {
+          include: ["packages/embeds/**/*.{test,spec}.{ts,js}"],
+          environment: "jsdom",
+        },
+      },
+    ]
+  : [
+      {
+        test: {
```

Comment:
Don't slowdown core tests for packaged code tests(which require build for all embed packages).